src/test/test_verify.cpp

```diff
@@ -424,8 +424,7 @@ TEST_SECTION("falco", "probe.o", "raw_tracepoint/signal_deliver")
 TEST_SECTION_REJECT_IF_STRICT("build", "mapoverflow.o", ".text")
 TEST_SECTION_REJECT_IF_STRICT("build", "mapunderflow.o", ".text")
 
-// Test uses global variables, which are not supported yet by the verifier.
-TEST_SECTION_FAIL("build", "global_variable.o", ".text")
+TEST_SECTION("build", "global_variable.o", ".text")
```

Comment:
Move this lilne down to around line 519 were the other `TEST_SECTION("build", ...)` lines are